src/apps/shared/filtering.py

```diff
@@ -0,0 +1,38 @@
+import operator
+
+__all__ = ["Filtering", "FilterField"]
+
+from sqlalchemy import Column
+
+lookups = {
+    "eq": operator.eq,
+    "neq": operator.ne,
+}
+
+
+class FilterField(object):
+    def __init__(self, field: Column, lookup: str = "eq"):
+        assert lookup is not None and isinstance(lookup, str)
+        self.field = field
+        self.lookup = lookup
+        self._lookup = lookups.get(lookup, lookups["eq"])
+
+    def generate_filter(self, val):
+        return self._lookup(self.field, val)
```

Comment:
In case the `lookup` field will be an empty string - the line 15 will not fail. Is it expected?